config/jobs/kubernetes/cloud-provider-gcp/cloud-provider-gcp-periodics.yaml

```diff
@@ -0,0 +1,33 @@
+periodics:
+- interval: 3h
+  cluster: k8s-infra-prow-build
+  name: ci-cloud-provider-gcp-conformance-latest
+  annotations:
+    testgrid-tab-name: Conformance - Cloud Provider GCP - master
+    testgrid-dashboards: provider-gcp-periodics
+    description: Runs conformance tests using kubetest2 against cloud-provider-gcp master on GCE
+  labels:
+    preset-service-account: "true"
+  spec:
+    containers:
+    - image: gcr.io/k8s-testimages/krte:v20200806-28035c4-1.16
+      command:
+      - wrapper.sh
+      args:
+      - "/bin/bash"
+      - "-c"
+      # TODO: Use published release tars for cloud-provider-gcp if/once they exist
+      - set -o errexit;
+        set -o nounset;
+        set -o pipefail;
+        set -o xtrace;
+        git clone https://github.com/kubernetes/cloud-provider-gcp;
```

Comment:
can we use `extra_refs` instead of manually git cloning.

Reply:
Periodics don't get access to `extra_refs`. I believe this is also indicated by the fact that there is no `clonerefs` containers as part of periodic job specs.

Reply:
I'm not sure if that's true. `extra_refs` is a [ProwJob field](https://godoc.org/k8s.io/test-infra/prow/apis/prowjobs/v1#ProwJobSpec) so it's independent of whether it's a presubmit or a periodic.
We use it in a bunch of jobs https://github.com/kubernetes/test-infra/blob/70c45962f9a72e8c8141243612b15e3b6aaf6141/config/jobs/kubernetes-sigs/kind/kind.yaml#L5-L9

Reply:
Hmm. I tried adding it to my periodic, regenerating it, and then looking for the repo and couldn't find it.

Reply:
you'll need `decorate: true`

Reply:
`decorate: true` solves this problem. Updated the PR to use `extra_refs` to get the `cloud-provider-gcp` repo.